client/my-sites/checkout/upsell-nudge/business-plan-upgrade-upsell/index.jsx

```diff
@@ -223,7 +223,7 @@ export class BusinessPlanUpgradeUpsell extends PureComponent {
 								) }
 							{ ! hasSevenDayRefundPeriod &&
 								translate(
-									'The good news is that you can upgrade your plan today and try the Business plan risk-free thanks to our {{b}}30-day money-back guarantee{{/b}}.',
+									'The good news is that you can upgrade your plan today and try the Business plan risk-free thanks to our {{b}}14-day money-back guarantee{{/b}}.',
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.